engine/src/state_chain_observer/client/extrinsic_api/signed/submission_watcher.rs

```diff
@@ -31,6 +31,7 @@ use crate::state_chain_observer::client::{
 	storage_api::{CheckBlockCompatibility, StorageApi},
 	SUBSTRATE_BEHAVIOUR,
 };
+use futures::StreamExt;
```

Comment:
Nit: is this only used in the test file? Might make sense to move the import there if so.

Reply:
It is required by` transaction_status_stream.next()`.